### DISCUSSION TITLE
Advice on styling with Tailwind

### DISCUSSION THREAD

**Discussion by timmyd87** [orchidjs/tom-select#75] [Q&A]
Hi Team,

Love this fork. Just after some advice on styling the input with tailwind. I've had a go using the base css file and applying some styles to the 'tomselected' class but im getting some funny behaviour with the original input showing in addition to the dropdown.

Is there a way to apply stying without it appearing on the original input? Maybe through the API?

**Comment by oyejorge** [comment]
Hmm, the "hidden" attribute is added to the original input so you might be overriding that with your CSS somehow. Hard to say without seeing it though. A JSFiddle or Codepen example would help.
